docs/doc_usage_cmd.md

```diff
@@ -21,7 +21,32 @@ To run DomainLab, the minimum necessary parameters are:
 ### Advanced Configuration
 
 - **Learning Rate (`--lr`):** Set the training learning rate.
-- **Regularization (`--gamma_reg`):** Weight of regularization loss.
+- **Regularization (`--gamma_reg`):** Sets the weight of the regularization
+ loss. This parameter can be configured either as
+ a single value applied to individual classes,
+ or using a dictionary to specify different
+ weights for different models and trainers.
+
+  - **Command Line Usage:**
+    - For a single value: `python script.py --gamma_reg=0.1`
+    - For multiple values: `python script.py --gamma_reg='default=0.1,dann=0.05,diva=0.2'`
+
+  - **YAML Configuration:**
+    - For a single value:
+
+      ```yaml
+      gamma_reg: 0.1
+      ```
+
+    - For different values:
+
+      ```yaml
+      gamma_reg:
+        dann: 0.05
+        diva: 0.2
+        default: 0.1 # value for every other instance
```

Comment:
so the idea here is only special gamma_reg for model dann and model diva?  for trainers only one gamma_reg is allowed?  e.g. if we have mhof-fishr-dann-diva, will this be allowed?

Reply:
@MatteoWohlrapp , forget about the above i removed diva because diva does not have a gamma hyperparameter due to its special implementation.

Reply:
can mldg also be specified here?

Reply:
No the idea is that you can specify it for all trainers and the models that support gamma regularization. Diva is actually false here, because only jigen and dann support it from the models. I changed the comment in the last commit, but I think every trainer and the two model mentioned above can be specified here.